docs/diagrams/ViewTagSequenceDiagramLogic.puml

```diff
@@ -26,36 +23,29 @@ activate TagCommand
 TagCommand --> AddressBookParser : command
 deactivate TagCommand
 
-AddressBookParser --> LogicManager : command
+AddressBookParser --> Logic : command
 deactivate AddressBookParser
 
-LogicManager -> TagCommand : execute()
+Logic -> TagCommand : execute()
 activate TagCommand
 
-TagCommand -> Model : getObservableTagList()
-activate Model
-
-Model --> TagCommand
-deactivate Model
-
 create CommandResult
 TagCommand -> CommandResult
 activate CommandResult
 
-CommandResult --> TagCommand : CommandResult
+CommandResult --> TagCommand : commandResult
 deactivate CommandResult
 
-'Hidden arrow to position the destroy marker below the end of the activation bar.
-CommandResult -[hidden]-> AddressBookParser
-destroy CommandResult
-
-TagCommand --> LogicManager : commandResult
+TagCommand --> Logic : commandResult
 deactivate TagCommand
 
 'Hidden arrow to position the destroy marker below the end of the activation bar.
-TagCommand -[hidden]-> AddressBookParser
+TagCommand -[hidden]-> Logic
 destroy TagCommand
```

Comment:
Why destroy here? The DeleteCommand example they originally gave us only destroys the Parser and deactivates the Command?

![image](https://user-images.githubusercontent.com/62977276/140600359-9a059e6a-89db-4bda-ab31-a6e14ceb6f8e.png)

Reply:
changed. btw should i shift reminder feature to j underneath the lesson management section?

Reply:
Yeah, I think can shift since it references lessons quite a bit